### PR TITLE
Correct Greater Nosoi Charm, and touch up REs

### PR DESCRIPTION
**Pull request by LebombJames**
The Greater Nosoi charm had an identical price and level to the standard one. Also condensed some of the effects into the item itself, and localised the RE labels.

### CODE REVIEW

File: packs/data/equipment.db/nosoi-charm.json [hunk 1]
```diff
@@ -35,7 +35,27 @@
...
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "label": "PF2E.SpecificRule.NosoiCharm.Performance",
+                "predicate": [
+                    {
+                        "or": [
+                            "action:play-an-instrument",
+                            "action:orate",
+                            "action:sing"
```
Comment (TikaelSol) [inline packs/data/equipment.db/nosoi-charm.json:47]:
```suggestion
                            "action:performance:play-an-instrument",
                            "action:performance:orate",
                            "action:performance:sing"
```

File: packs/data/equipment.db/nosoi-charm-greater.json [hunk 2]
```diff
@@ -31,11 +31,37 @@
...
+            },
+            {
+                "key": "FlatModifier",
+                "label": "PF2E.SpecificRule.NosoiCharm.Performance",
+                "predicate": [
+                    {
+                        "or": [
+                            "action:play-an-instrument",
+                            "action:orate",
+                            "action:sing"
```
Comment (TikaelSol) [inline packs/data/equipment.db/nosoi-charm-greater.json:53]:
```suggestion
                            "action:performance:play-an-instrument",
                            "action:performance:orate",
                            "action:performance:sing"
```

File: packs/data/equipment-effects.db/effect-nosoi-charm-diplomacy.json [hunk 3]
```diff
@@ -19,9 +19,9 @@
         "rules": [
             {
                 "key": "FlatModifier",
-                "label": "Nosoi Charm (Make A request)",
+                "label": "PF2E.SpecificRule.NosoiCharm.Request",
```
Comment (TikaelSol) [inline packs/data/equipment-effects.db/effect-nosoi-charm-diplomacy.json:22]:
```suggestion
                "label": "PF2E.SpecificRule.Equipment.NosoiCharm.RequestLabel",
```
Let's move this to an equipment subsection in there. Ideally I would like to have started with more organization in the first place but now's a good time to start too. If you rebase this branch you'll see an equipment section was made recently. I'm also not convinced that these need labels but if they are localized it's not going to bother me much.

File: packs/data/equipment-effects.db/effect-nosoi-charm-greater-lifesense.json [hunk 4]
```diff
@@ -1,45 +0,0 @@
-{
-    "_id": "0rkg7SVzs2AetUXw",
-    "img": "systems/pf2e/icons/equipment/held-items/nosoi-charm.webp",
-    "name": "Effect: Nosoi Charm (Greater) (Lifesense)",
-    "system": {
-        "badge": null,
-        "description": {
-            "value": "<p>You gain lifesense as an imprecise sense with a range of 30 feet as long as you are holding the charm, and do not need to activate it. This allows you to sense the life force that animates living creatures and the perverse force that animates the dead, though you can't distinguish between the two.</p>"
```
Comment (TikaelSol) [inline packs/data/equipment-effects.db/effect-nosoi-charm-greater-lifesense.json:8]: I assume these REs have been moved to the item itself?

Comment (LebombJames) [reply packs/data/equipment-effects.db/effect-nosoi-charm-greater-lifesense.json:8]: They have been, yes

**Pull request merged**